life/modernized/abel-mod.adoc

```diff
@@ -1443,7 +1443,7 @@ and have laboured through almost all the meetings belonging to this Yearly meeti
 In a few days I shall travel towards New York state, taking in my way,
 Pomfret and Kenford, towards New Milford and Oblong; then,
 taking the meetings of Friends on to Danby and Ferrisburg, and return back to Granville,
-Queensbury, and down the west side of the river to Marlborouofh,--the meetings thereaway,
+Queensbury, and down the west side of the river to Marlborouogh,--the meetings thereaway,
```

Comment:
I think this should be "Marlborough" unless I'm mistaken.
I'm spying on you lol